src/tinyexpr.cpp

```diff
@@ -182,6 +182,10 @@ static constexpr double bit_xor(double a, double b) {
     return static_cast<double>(static_cast<long long>(a) ^ static_cast<long long>(b));
 }
 
+static double max(double a, double b) { return std::max(a,b); }
+
+static double min(double a, double b) { return std::min(a,b); }
```

Comment:
`clang-format` should add a space between arguments :detective:

Reply:
My bad, I don't actually have `clang` on my machine 😄